src/pages/layout/Search/SearchInput.tsx

```diff
@@ -0,0 +1,45 @@
+import React from "react";
+import {
+  FormControl,
+  InputAdornment,
+  SvgIcon,
+  SvgIconProps,
+  TextField,
+} from "@mui/material";
+
+function SearchIcon(props: SvgIconProps) {
```

Comment:
why not using a search icon...? https://mui.com/material-ui/material-icons/?query=search&selected=Search

Reply:
Good question, I'm just reusing what we've been using. Let me make the change.